packages/components/src/DataList/DataList.css

```diff
@@ -175,7 +175,7 @@
   transition: opacity var(--transition-properties);
   --transition-properties: var(--timing-quick) ease-in-out;
 
-  @media (--medium-screens-and-up) and (any-pointer: fine) {
+  @media (any-pointer: fine) {
```

Comment:
@chris-at-jobber I updated it so that there is no more breakpoint.

Based on mdn docs if the media query isn't support it will return false (aka always be visible)
<img width="824" alt="image" src="https://github.com/user-attachments/assets/6763ca10-575c-4e33-9210-b7ceb310bc9a">
To test this you can rewrite this so it has a typo `@media (an-pointer: fine)` to simulate an unknown media query and the checkbox will always be displayed.
CC: @taylorvnoj